gui/qt/paytoedit.py

```diff
@@ -37,6 +37,10 @@
 from . import util
 
 RE_ALIAS = '^(.*?)\s*\<([0-9A-Za-z:]{26,})\>$'
+RE_COINTEXT = r'^cointext:([-+ 0-9]+)\s*$'
```

Comment:
You might want to include brackets here, they are commonly used in some countries and can just be omitted.

Reply:
I didn't know that.. so which braces? `[` ? `{`? `(`?

Reply:
Just include ( and ), they are often used to group numbers. While they are not the E.164 standard, neither is dash (-).

Reply:
Ok, will do. thanks. :)

Reply:
62420804bfee8a671b12e273d35f2bc2ca73763d